R/splitOn.R

```diff
@@ -13,25 +13,31 @@
 #'   Split by cols is not encouraged, since this is not compatible with 
 #'   storing the results in \code{altExps}.
 #'
-#' @param keep_reducedDims \code{TRUE} or \code{FALSE}: Should the
+#' @param keep.reduceddims \code{TRUE} or \code{FALSE}: Should the
```

Comment:
`reduceddims` or `reddims`? (more broad question)

Reply:
scater package uses dimred (for instance https://rdrr.io/bioc/scater/man/plotReducedDim.html)

We could use dimred also

Reply:
Hmm the downside is that the name doesnt match our reducedDim as well then. That might be confusing?

Reply:
It does not match SCE's reducedDim either. I believe the parameter name comes from dimension reduction --> dimred

It is true that it does not that clearly state the slot. Also, we do not have to use same names as in scater. But one thing to note is that we already use dimred (for example in getMediation function)

Reply:
would be good to use same name across pkgs

Reply:
dimred is widely used in single-cell packages (they do not use reddim in parameter names). I would go with keep.reduceddim or keep.dimred

I prefer keep.dimred